data/human/human missions.txt

```diff
@@ -4077,3 +4077,98 @@ mission "Laborer Payment"
 				decline
+
+
+
+mission "Block Hai Wormhole Warning"
+	landing
+	invisible
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		has "chosen sides"
+		not "main plot completed"
+	on offer
+		fail
+
+mission "Hai Wormhole Warning"
+	minor
+	source
+		government "Republic"
+		not attributes "deep"
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		not "Block Hai Wormhole Warning: offered"
+		not "Wanderers: Alpha Surveillance E: offered"
+	on offer
+		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
+		conversation
+			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
+			choice
+				`	"What about?"`
+					goto what
+				`	"Okay."`
+					goto frigate
+				`	"Sorry, I'm really busy right now."`
+			`	"No, I'm sorry, but this isn't a matter of whether you're busy or not right now. Come with me."`
+				goto frigate
+			label what
+			`	"I did say in private, did I not?" he says, once again motioning to the Frigate.`
+			label frigate
+			`	The man puts a hand on your shoulder and begins walking you toward the ship. As you approach, you note that the man's uniform doesn't have any Naval markings that you recognize.`
+			`	You enter the ship with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
+			choice
+				`	(Take a seat.)`
+					goto seat
+				`	"No, I like to stand."`
+			`	"Suit yourself," the officer says with a shrug while taking a seat himself.`
+				goto questioning
+			label seat
+			`	You choose a table to sit at and the officer sits across from you.`
+			label questioning
+			`	"So what is it that you brought me here for?" you ask.`
+			`	The officer folds his hands together. "It has come to the Republic's attention that you've made contact with an alien species located to the north of the Republic. Is this correct?"`
+			choice
+				`	"It is. Is that a problem?"`
+					goto yes
+				`	"Perhaps. Why do you ask?"`
+					goto maybe
+				`	"Aliens? I don't know of any aliens up there."`
+			`	"Don't play coy with me, <last>. We are aware that you've made contact with the Hai.`
+				goto secret
+			label maybe
+			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' We are aware that you've made contact with the Hai.`
+				goto secret
+			label yes
+			`	"There is no problem, <last>, but we are aware that you've made contact with the Hai.`
```

Comment:
Following the other two options, this phrase makes sense, but here it seems a bit odd.
"The Republic has learned that you've made contact with some alients to the north. Is this correct?"
"It is, is that a problem?"
"No problem, but we are aware that you've made contact with the Hai."
Dropping the question in our answer:
"It is."
"We are aware that you've made contact with the Hai."
My response would be something like:
"Yeah, you already told me that, and I just told you that I have."
Maybe if he thought we might be talking about the Wanderers, but there's no indication anywhere else that that's the case.
```suggestion
			`	"There is no problem, <last>, I just want to make sure you also know about an arrangement the Republic has with the Hai.`
			`	"The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we will not allow this existence to be widely shared."
				goto question
```